edc-extensions/postgresql-migration/src/main/resources/org/eclipse/tractusx/edc/postgresql/migration/transferprocess/V0_0_6__Snapshot_20230109_Update.sql

```diff
@@ -9,7 +9,10 @@
 --
 --  Contributors:
 --       ZF Friedrichshafen AG - Add column for the dynamically HTTP Receiver callback endpoints
+--       Mercedes-Benz Tech Innovation GmbH - Add default properties
 --
 
 -- add column
 ALTER TABLE edc_transfer_process ADD COLUMN transferprocess_properties TEXT;
+-- set default value
+ALTER TABLE edc_transfer_process ADD CONSTRAINT transferprocess_properties DEFAULT "{}";
```

Comment:
Missing empty line at EOF
```suggestion
ALTER TABLE edc_transfer_process ADD CONSTRAINT transferprocess_properties DEFAULT "{}";

```